pkgs/development/compilers/llvm/11/clang/default.nix

```
@@ -8,12 +8,12 @@ let
     pname = "clang";
     inherit version;
 
-    src = fetch "clang" "02ajkij85966vd150iy246mv16dsaph1kfi0y8wnncp8w6nar5hg";
+    src = fetch "clang" "0kab4zmkxffg98a3rx95756jlwhxflalin5w05g1anpwxv175xbk";
     inherit clang-tools-extra_src;
 
     unpackPhase = ''
       unpackFile $src
-      mv clang-${version}* clang
+      mv clang-* clang
```

```suggestion
      mv clang-${release_version}* clang
```

Probably doesn't matter but from your commit message:
> it was looking for `clang-11.0.1-rc2.src` but actually it is `clang-11.0.1rc2.src`

this should also work and would be consistent with your other change below.